orchestration/src/main/java/com/sap/ai/sdk/orchestration/client/model/MaskingProviderConfig.java

```diff
@@ -12,395 +12,17 @@
-    sb.append("    method: ").append(toIndentedString(method)).append("\n");
-    sb.append("    entities: ").append(toIndentedString(entities)).append("\n");
-    cloudSdkCustomFields.forEach(
-        (k, v) ->
-            sb.append("    ").append(k).append(": ").append(toIndentedString(v)).append("\n"));
-    sb.append("}");
-    return sb.toString();
-  }
-
-  /**
-   * Convert the given object to string with each line indented by 4 spaces (except the first line).
-   */
-  private String toIndentedString(final java.lang.Object o) {
-    if (o == null) {
-      return "null";
-    }
-    return o.toString().replace("\n", "\n    ");
-  }
-
-  /**
-   * Create a type-safe, fluent-api builder object to construct a new {@link MaskingProviderConfig}
-   * instance with all required arguments.
-   */
-  public static Builder create() {
-    return (type) ->
-        (method) ->
-            (entities) -> new MaskingProviderConfig().type(type).method(method).entities(entities);
-  }
-
-  /** Builder helper class. */
-  public interface Builder {
-    /**
-     * Set the type of this {@link MaskingProviderConfig} instance.
-     *
-     * @param type Type of masking service provider
-     * @return The MaskingProviderConfig builder.
-     */
-    Builder1 type(@Nonnull final TypeEnum type);
-  }
-
-  /** Builder helper class. */
-  public interface Builder1 {
-    /**
-     * Set the method of this {@link MaskingProviderConfig} instance.
-     *
-     * @param method Type of masking method to be used
-     * @return The MaskingProviderConfig builder.
-     */
-    Builder2 method(@Nonnull final MethodEnum method);
-  }
-
-  /** Builder helper class. */
-  public interface Builder2 {
-    /**
-     * Set the entities of this {@link MaskingProviderConfig} instance.
-     *
-     * @param entities List of entities to be masked
-     * @return The MaskingProviderConfig instance.
-     */
-    MaskingProviderConfig entities(@Nonnull final List<DPIEntityConfig> entities);
-
-    /**
-     * Set the entities of this {@link MaskingProviderConfig} instance.
-     *
-     * @param entities List of entities to be masked
-     * @return The MaskingProviderConfig instance.
-     */
-    default MaskingProviderConfig entities(@Nonnull final DPIEntityConfig... entities) {
-      return entities(Arrays.asList(entities));
-    }
-  }
-}
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@javax.annotation.Generated(
+    value = "org.openapitools.codegen.languages.JavaClientCodegen",
+    date = "2024-11-08T10:43:10.628772+01:00[Europe/Berlin]",
+    comments = "Generator version: 7.9.0")
+@JsonIgnoreProperties(
+    value = "", // ignore manually set , it will be automatically generated by Jackson during
+    // serialization
+    allowSetters = true // allows the  to be set during deserialization
+    )
+@JsonTypeInfo(use = JsonTypeInfo.Id.DEDUCTION)
+public interface MaskingProviderConfig {}
```

Comment:
(Question)

This is custom / hand-crafted code(?)
How do you protect this class from being overwritten when the generator runs next time?

Reply:
> This is custom / hand-crafted code(?)

_"All interface types are hand-crafted"_

> How do you protect this class from being overwritten when the generator runs next time?

_"see openapi generator ignore file"_